### PR TITLE
Fix Beego_test and add fiberEngine Test.

### PR DESCRIPTION
**Pull request by ralic**
Revised PR. To add new feature to support fiber engine. (https://github.com/gofiber)

Original PR: https://github.com/appleboy/gofight/pull/83

### CODE REVIEW

File: _example/beego_test.go [hunk 1]
```diff
@@ -2,26 +2,25 @@ package example
...
 	"fmt"
-	"net/http"
-	"testing"
-
 	"github.com/appleboy/gofight/v2"
 	"github.com/astaxie/beego"
 	"github.com/stretchr/testify/assert"
+	"net/http"
+	"testing"
 )
```
Comment (appleboy) [inline _example/beego_test.go:10]: revert these changes.

File: go.mod [hunk 2]
```diff
@@ -2,4 +2,23 @@ module github.com/appleboy/gofight/v2
...
+	github.com/julienschmidt/httprouter v1.2.0
+	github.com/klauspost/compress v1.11.7 // indirect
+	github.com/kr/text v0.2.0 // indirect
+	github.com/labstack/echo/v4 v4.1.17
+	github.com/stretchr/testify v1.7.0
+	github.com/valyala/fasthttp v1.19.0 // indirect
+	golang.org/x/sys v0.0.0-20210113181707-4bcb84eeeb78 // indirect
+	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c // indirect
+	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
+)
```
Comment (appleboy) [inline go.mod:24]: revert these changes.

File: gofight.go [hunk 3]
```diff
@@ -52,7 +52,9 @@ package gofight
 import (
 	"bytes"
 	"encoding/json"
+	"github.com/gofiber/fiber/v2"
```
Comment (appleboy) [inline gofight.go:55]: move github.com to the bottom and add an empty line in the front.

**Review by appleboy — CHANGES_REQUESTED** [review]

File: go.mod (hunk 2 above)
Comment (ralic) [reply go.mod:9]: 1.To add fiber support might have to add fiber itself into the go module file, since fiber server is based on the fasthttp, which is different from the golang's original net/http package. 
Please see following reference for performance benchmarking.
fasthttp: [(https://github.com/valyala/fasthttp)](https://github.com/valyala/fasthttp)
fiber: [(https://github.com/gofiber/fiber)](https://github.com/gofiber/fiber)
2. It would be better to keep the version of beego used in testing indirectly to keep the test working.

File: gofight.go (hunk 3 above)
Comment (ralic) [reply gofight.go]: Done

File: _example/beego_test.go (hunk 1 above)
Comment (ralic) [reply _example/beego_test.go]: Done

File: go.mod (hunk 2 above)
Comment (appleboy) [reply go.mod:9]: Yes, I know your point. I remove all packages from `go.mod` since renaming the `example` to `_example`. So we don't need all the framework package if you want to use the package.

Comment (appleboy) [reply go.mod:9]: See the reference: https://github.com/appleboy/gofight/pull/81

Comment (appleboy) [reply go.mod:9]: Any feedbacks?

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
I remove the fiber package since fiber doesn't support `ServeHTTP`.
